tests/scripts/test_get_poetry.py

```diff
@@ -0,0 +1,72 @@
+import os
+import shutil
+
+
+try:
+    # Python 3.5+
```

Comment:
You can drop compatibility changes as we no longer test under py2 or py35.

Reply:
Well that simplifies things!